src/engines/kotor/module.cpp

```diff
@@ -519,6 +519,14 @@ void Module::setReturnEnabled(bool enabled) {
 	_ingame->setReturnEnabled(enabled);
 }
 
+void Module::setGlobalNumber(const Common::UString &id, int value) {
+	_globalNumbers[id] = value;
+}
+
+int Module::getGlobalNumber(const Common::UString &id) {
+	return _globalNumbers[id];
+}
```

Comment:
Make that method const, by using `_globalNumbers.find()` instead of `operator[]`. This also means that you need to explictly state the default value (which would be 0, I suppose?) when the id key is not within the map.